src/main/scala/rps/Game.scala

```diff
@@ -2,28 +2,36 @@ package rps
 
 import scala.util.Random
 
-object Moves {
-  sealed class Move(val id: String, val name: String)
+sealed trait Move
+case object Rock extends Move
```

Comment:
good, but it's good practice to put the cases inside a companion object for namespacing:

```scala
sealed trait Move
object Move {
  case object Rock extends Move
  // ...
}

Reply:
I'm not sure how this happened but I think you commented on an old version of the commit, it's my fault as I `amended` some changes after pushing the commit... mea culpa 😓